sns/parity_math.c

```diff
@@ -86,37 +86,37 @@ static void fail_idx_xor_recover(struct m0_parity_math *math,
 				 struct m0_buf *parity,
 				 const uint32_t failure_index);
 
-/* Below are some functions which has two separate implementations for using
- * Galois and Intel ISA library. At a time any one of the function is
- * called depending on whether Intel ISA library is present or not. If
- * Intel ISA library is present, it will use implementation specific to Intel
- * ISA and use Intel ISA APIs for encoding and recovery, else it will use
- * Galois library arithmetic functions.
+/* Below are some functions which has two separate implementations for user
+ * space and kernel space. Intel ISA library is used in user space only. Hence
+ * implementation of these functions in kernel space are empty for now and will
+ * be removed once kernel space compilation is removed. The earlier
+ * implementation of Reed-Solomon algorithm which uses Galois or Intel ISA
+ * arithmetic function is currently commented under NA_FOR_INTEL_ISA macro.
  */
 
 /**
  * This function initialize fields required to use Reed Solomon algorithm.
- * It has two separate implementations for using Galois and Intel ISA library.
+ * It has two separate implementations for user space and kernel space for now.
```

Comment:
You do not need the comment "It has two separate implementations for user space and kernel space for now." because in the above comment on lines 89-94 you have already mentioned about 2 implementations.

Same changes for comments below till line 189.

Reply:
Done